src/github3/models.py

```diff
@@ -54,6 +54,8 @@ def _update_attributes(self, json):
 
     def __getattr__(self, attribute):
         """Proxy access to stored JSON."""
+        if attribute == '_json_data':
+            raise AttributeError(attribute)
```

Comment:
We could also do:

```py
json_data = super(GitHubCore, self).__getattr__(self, '__json_data')
if attribute not in json_data:
    raise AttributeError(attribute)
value = json_data.get(attribute)
setattr(self, attribute, value)
return value
```

`object.__getattr__` could be used instead of `super()` too but either way, we're bypassing the potentiality for recursion here.

Reply:
Also using `object.__getattr__` is more common to see and less arcane.